tox.ini

```diff
@@ -1,26 +1,26 @@
 [tox]
-envlist = lint, test
+envlist = py36-lint, py36-test
```

Comment:
There's no way to get Travis to play nice with the original names?

Reply:
Maybe there is. Feel free to try, I'm just happy I got it to work.
Plus having this format probably prepares us for supporting other Python versions, so I doubt we'll be able to keep it the way it was.